### DISCUSSION TITLE
Integrating multiomic data (CITE-Seq + ATAC-seq)

### DISCUSSION THREAD

**Discussion by markphillippebworth** [GreenleafLab/ArchR#736] [Feature Requests]
[Hello!]

Is there a way to add ADT data to an arrow file? There are several multiomic datasets that now merge CITE-seq and ATAC-seq, and it would be great to have protein data to overlay. 

Additionally, there's a new multiomic technique that combines ADT, ATAC, and RNA measurements (referred to as   
[TEA-seq](https://elifesciences.org/articles/63632) ). It would be amazing if we could run LSI on all three and then generate combined dims. 

There have been attempts at this with Seurat, but I'm having issues saving and processes datasets above 100K cells, and I would love to be able to use ArchR's data handling to generate large scale references easily. 

Thanks!
-MP

**Comment by rcorces** [comment]
Theoretically you can do this by treating the CITE-seq as if it were RNA-seq and using the currently available functionality for ATAC+RNA multiome datasets. But this would be a bit of a hack job. We dont have support for all three modalities yet and this will not be quick to come out because we are searching for additional hands to continue active development.

**Reply by markphillippebworth** [reply]
Gotcha, thank you for letting me know! I'll try to play around with it sometime. I'm finding that ArchR LSI iteration creates much better 3-way WNN graphs than Signac's LSI implementation. I'm looking forward to transfering all my analysis over to ArchR from Seuat, when you can get this working. I hope you're able to get some good people to continue this!

**Reply by connorrogerson** [reply]
Did you make any progress on this? I have a multiome dataset (so scATAC and scRNA) and a CITE-seq style dataset. I want to import these 3rd modality counts as another matrix. My first guess was to use addGeneExpressionMatrix and name the matrix something else but that doesn't seem to be an option. I'm all ears!
